docs/src/main/asciidoc/firestore.adoc

```diff
@@ -146,6 +146,13 @@ Cloud Firestore only supports filter components joined by AND, and the following
 * `less than`
 * `is null`
 
+Operations expecting List arguments:
```

Comment:
```suggestion
Operations expecting `List` arguments:
```
Also, I probably would just join with the list above. I think people understand that `in` and `contains` requires lists.

Reply:
Can you apply this little fix?